test.properties

```diff
@@ -8567,6 +8567,49 @@
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][translation]=\
         (testray.main.component.name ~ "Translations Management")
 
+    #
+    # UM
+    #
+
+    test.batch.class.names.includes[um]=\
+        **/account/**/*Test.java,\
+        **/address-test/**/*Test.java,\
```

Comment:
My only concern is that I'm unsure if the **address** component belongs to our team.